coalib/coala.py

```diff
@@ -74,7 +74,7 @@ def main():
     if args.non_interactive:
         return mode_non_interactive(console_printer, args)
 
-    if args.format:
+    if args.format or args.ci:
         return mode_format()
 
```

Comment:
`--ci` and `--format` are not same. That is why adding args.ci on line 77 breaks stuff.
`--ci` and `--non-interactive` are same. Refer `coalib/parser/DefaultArgParser`. 
Make required changes.